library/src/main/java/org/xmtp/android/library/libxmtp/DisappearingMessageSettings.kt

```diff
@@ -0,0 +1,14 @@
+package org.xmtp.android.library.libxmtp
+
+import uniffi.xmtpv3.FfiMessageDisappearingSettings
+
+class DisappearingMessageSettings(
+    val disappearStartingAtNs: Long,
+    val disappearDurationInNs: Long,
```

Comment:
second var name seems confusing in my opinion. maybe `retentionDurationInNs` is clearer?

Reply:
What about `disappearingInNS`? Or `disappearingIntervalNs`?